lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -0,0 +1,289 @@
+            return $m->reference_sequence_build;
+        }
+    }
+}
+
+#Filter out putative false-positive SNVs
+sub filter_snvs {
+    my $self = shift;
+    my $somvar_build = shift;
+    my $snv_prefix = shift;
+    my ($tumor_bam, $normal_bam) =
+    $self->get_tumor_normal_bam($somvar_build);
+    my $refbuild = $self->reference_build($somvar_build);
+    my $ref_fa = $refbuild->full_consensus_path('fa');
+    my $filter = Genome::Model::Tools::Varscan::SomaticFilterWorkflow->
+    create(
+        outdir => $self->outdir,
+        tumor_bam => $tumor_bam,
+        normal_bam => $normal_bam,
+        prefix => $snv_prefix,
+        reference => $ref_fa,
+        bamrc_version => $self->bamrc_version,
+    );
+    $filter->execute();
+}
+
+#Combine germline, LOH calls and sort them
+sub combine_sort_snvs {
+    my $self = shift;
+    my $snv_prefix = shift;
+    my $snv_combined = $snv_prefix . ".combined";
+    my $snv_combined_sorted = $snv_prefix . ".combined.sorted";
+    my @germline = glob $snv_prefix . "*.formatted.LOH.hc.filtered";
+    my @loh = glob $snv_prefix . "*.formatted.Germline.hc.filtered";
+    unless(scalar @germline and scalar @loh) {
+        die $self->error_message("Unable to find germline or
+            loh filtered results");
+    }
+    Genome::Sys->cat(
+            input_files => [@germline, @loh],
+            output_file => $snv_combined
+            );
+    my $sort = Genome::Model::Tools::Capture::SortByChrPos->
+    create(
+        input_file => $snv_combined,
+        output_file => $snv_combined_sorted,
+    );
+    $sort->execute;
+    return $snv_combined_sorted;
+}
+
+#Segment the combined calls
+sub segment_loh {
+    my $self = shift;
+    my $snvs_combined_sorted = shift;
+    my $loh_basename = shift;
+    my $segmenter = Genome::Model::Tools::Varscan::LohSegments->
+    create(
+        variant_file => $snvs_combined_sorted,
+        output_basename => $loh_basename,
+    );
+    $segmenter->execute();
+}
+
+#Filter out LOH segments without sufficient support
+sub filter_loh {
+    my $self = shift;
+    my $loh_basename = shift;
+    my $min_segmean = $self->min_segmean;
+    my $minprobes = $self->minprobes;
+    my $loh_segments =
+    $loh_basename . ".segments.cbs";
+    my $loh_segments_filtered = $loh_segments . ".filtered";
+    unless(-e $loh_segments) {
+        die $self->error_message("Unable to find loh file $loh_segments");
+    }
+    my $awk_filter =
+    "awk '\$4 > $minprobes && \$5 >= $min_segmean' $loh_segments " .
+        "> $loh_segments_filtered";
+    Genome::Sys->shellcmd(cmd => $awk_filter);
+}
+
+#Get rid of intermediate files
+sub cleanup {
+    my $self = shift;
+    my $unwanted;
+    foreach my $suffix ("*.Somatic*", "*.readcounts", "*.hc", "*.LOH",
+        "*.lc", "*.Germline", "*.removed", "*.formatted",
+        "*.hc.err", "*.other") {
+        my $unwanted = File::Spec->join($self->outdir, $suffix);
+        if(-e $unwanted) {
```

Comment:
This will only match if the filenames literally contain `*`.  You'd need something like:
```perl
for my $file (glob($unwanted))  {
    unlink $file;
}
```